brooklinevoiceapp/mycity/intents/polling_stations_intent.py

```diff
@@ -0,0 +1,109 @@
+""" Intent for responding to polling station requests """
+import logging
+from mycity.mycity_response_data_model import MyCityResponseDataModel
+from mycity.intents import intent_constants
+from mycity.utils.address_utils import set_address_in_session
+from mycity.utils.brookline_arcgis_api_utils import get_polling_locations_json
+
+
+LOGGER = logging.getLogger(__name__)
+
+# User facing strings
+CARD_TITLE_POLLING = "Polling Locations"
+OUTPUT_SPEECH_TEMPLATE = "The {} polling station in {} is located at {}. "
+
+# Strings used in parsing json data returned by server
+FEATURES_PATH = "features"
+ATTRIBUTES_PATH = "attributes"
+NAME_PATH = "NAME"
+PRECINCT_PATH = "PRECINCT"
+ADDR_PATH = "FULLADD"
+
+# Request data model strings
+NUMBER_LOCATIONS_SLOT_NAME = "number_requests"
+MAX_LOCATIONS = 10
+DEFAULT_LOCATIONS = 3
+
+def get_polling_location_info(mycity_request):
+    """
+    Generates a response to a polling location request
+
+    :param mycity_request: MyCityRequestDataModel containing the user request
+    :return: MyCityResponseDataModel containing the speech to return to the user
+    """
+    LOGGER.debug('Getting polling location information')
+
+    response = MyCityResponseDataModel()
+    set_address_in_session(mycity_request)
+    current_address = \
+            mycity_request.session_attributes.get(intent_constants.CURRENT_ADDRESS_KEY)
+    if current_address is None:
+        # Delegate to the Alexa interaction model for getting the user address
+        LOGGER.debug('Requesting user address.')
+        response.dialog_directive = "Delegate"
```

Comment:
We could alternatively use [auto delegation](https://developer.amazon.com/docs/custom-skills/delegate-dialog-to-alexa.html#automatically-delegate-simple-dialogs-to-alexa)